### ISSUE TITLE
Docs: define "production-ready"

### ISSUE THREAD

**Issue by moltar** [awslabs/project-lakechain#23]
### What were you searching in the docs?

https://awslabs.github.io/project-lakechain/general/faq/

> Is Project Lakechain production-ready?

### Is this related to an existing documentation section?

https://awslabs.github.io/project-lakechain/general/faq/

### How can we improve?

```Markdown
It is not clear what "production-ready" means.

What makes this project not production-ready?

What steps are required to make it production-ready?
```

### Acknowledgment

- [X] I understand the final update might be different from my proposed suggestion, or refused.

**Comment by HQarroum** [comment]
Hi @moltar! You're right, we should make it clearer whether or not customers can assume they can use Project Lakechain in production workloads, and what a production workload entails.

First, I would say that the definition of a production workload for us is any workload that has a business critical nature, where breaking the software API, introducing new bugs, lacking full test coverage to identify regressions, and not being as optimized as we'd like may lead to business related impacts.

Here's the rationale on why we've put this statement in the FAQ.

- The API associated with the middlewares and the core framework is currently unstable, which means that we *will* break APIs until we hit **1.0.0**, which may create frustration for customers that would already run the project in a production workload, and suddenly have to update the code during an update to update their CDK stacks, or worse, have to roll-back end re-deploy.
- Even if we do have automated testing in place in various places, it is currently not enough to protect customer workloads. We aim to soon have at least 80% code coverage across the key parts of the project.
- We're in Alpha preview, which means we're both evolving the project fast, but are also looking for feedbacks (such as yours!) from our customers to head the project to **1.0.0** where we'll consider it "production-ready" and will have the processes in place on our side to support customers on a day-to-day.

As your feedback makes a lot of sense, here's how I'd propose to update the FAQ.

"**Is Project Lakechain production-ready?**

No, Project Lakechain is currently released under Alpha preview, and is intended for rapid prototyping and experimentation only. While the middlewares and the core framework are being evolved fast as we receive feedbacks and strengthen our [Roadmap](https://github.com/orgs/awslabs/projects/153), it also means that their APIs are considered unstable until we reach **1.0.0**.

We take our customer workloads very seriously, and we're currently working towards a Beta release that will contain the core features we intend to ship, full test coverage across the framework, and more stable APIs overall. We're continuously looking for feedbacks to improve Project Lakechain, and we're looking to hear from you! If you encounter bugs, regressions, or have feature requests, we'd be very happy to [listen for your feedbacks](https://github.com/awslabs/project-lakechain/issues/new/choose)."

Let me know what you think, feel free to edit in case it's not clear enough.

**Comment by moltar** [comment]
That certainly clears up, at least for me, and I understand this enough to make an educated choice. Thank you. 🙏

**Comment by HQarroum** [comment]
Merged in #257fda2.
